supervisely/project/project_meta.py

```diff
@@ -347,8 +347,16 @@ def from_json(cls, data: Dict) -> ProjectMeta:
             obj_tag_metas = TagMetaCollection.from_json(obj_tag_metas_json)
             tag_metas = _merge_img_obj_tag_metas(img_tag_metas, obj_tag_metas)
 
+        try:
+            obj_classes = ObjClassCollection.from_json(data[ProjectMetaJsonFields.OBJ_CLASSES])
+        except KeyError:
+            raise KeyError(
+                f"Project meta JSON does not contain the {ProjectMetaJsonFields.OBJ_CLASSES} section. "
```

Comment:
key "XX" with the list of annotation classes not found in meta.json file.  Check the annotation format documentation at 
https://developer.supervisely.com/api-references/supervisely-annotation-json-format/project-classes-and-tags